internal/errors/pgerror.go

```diff
@@ -0,0 +1,21 @@
+package errors
+
+import (
+	"errors"
+
+	"github.com/jackc/pgx/v5/pgconn"
+)
+
+// ErrPgCode refers to https://www.postgresql.org/docs/current/errcodes-appendix.html
+
+const (
+	ErrPgCodeUniqueConstraints = "23505"
+)
+
+func IsPgErrorCode(err error, code string) bool {
```

Comment:
This should not be part of errors. It will now couple postgres to everywhere error is handled.

Reply:
@sbchaos should I move it into this package instead `internal/store/postgres` ?

Reply:
Sure, you can move it to postgres module.